packages/react-combobox/src/components/Listbox/useListbox.ts

```diff
@@ -11,18 +16,84 @@ import type { ListboxProps, ListboxState } from './Listbox.types';
  * @param props - props from this instance of Listbox
  * @param ref - reference to root HTMLElement of Listbox
  */
-export const useListbox_unstable = (props: ListboxProps, ref: React.Ref<HTMLElement>): ListboxState => {
+export const useListbox_unstable = (
```

Comment:
Is the intention for the `Listbox` component that it can be used separately from `Combobox` ?

Reply:
Yup, it can be used separately 👍